### PR TITLE
Output location updates to align with PR#5388 in azure-sdk-for-python

### PR DESCRIPTION
**Pull request by scbedd**
[Origin PR for changes](https://github.com/Azure/azure-sdk-for-python/pull/5388)

This PR will update the output location for swagger to reflect the newly moved files.

@lmazuel for pr and approval.

### CODE REVIEW

File: specification/operationalinsights/data-plane/readme.md [hunk 1]
```diff
@@ -105,12 +105,12 @@ directive:
 ``` yaml $(python) && $(python-mode) == 'update'
 python:
   no-namespace-folders: true
-  output-folder: $(python-sdks-folder)/azure-loganalytics/azure/loganalytics
+  output-folder: $(python-sdks-folder)/operationalinsights/azure-loganalytics/azure/loganalytics
```
Comment (lmazuel) [inline specification/operationalinsights/data-plane/readme.md]: "loganalytics" is the official name, operationalinsights is the RP name only (not visible to customer)

File: specification/resourcegraph/resource-manager/readme.python.md [hunk 2]
```diff
@@ -18,10 +18,10 @@ python:
 ``` yaml $(python) && $(python-mode) == 'update'
 python:
   no-namespace-folders: true
-  output-folder: $(python-sdks-folder)/azure-mgmt-resourcegraph/azure/mgmt/resourcegraph
+  output-folder: $(python-sdks-folder)/resourcegraph/azure-mgmt-resourcegraph/azure/mgmt/resourcegraph
```
Comment (lmazuel) [inline specification/resourcegraph/resource-manager/readme.python.md]: Fine to me, but to save a folder you can put it in "resources"

File: specification/operationalinsights/data-plane/readme.md (hunk 1 above)
Comment (scbedd) [reply specification/operationalinsights/data-plane/readme.md]: I hear you. So far we've been standardizing on following the spec repo for service names. (I actually originally moved under `loganalytics` , before Wes called me out on it to align with the other repos )

Comment (lmazuel) [reply specification/operationalinsights/data-plane/readme.md]: I oppose strongly on this one, our repo is the face to customer and should not show our internal organisation. Operationalinsights doesn't exist for customer, and we shall not show it in the repo.
Ready to discuss with Wes on it if necessary.

**Pull request merged**